.github/workflows/ja.yaml

```diff
@@ -0,0 +1,53 @@
+name: JA
+
+on:
+  push:
+    paths:
+      - ".github/workflows/ja.yaml"
+      - "site/ja/**"
+      - "tools/ja/**"
+
+jobs:
+  proofreading-ja:
+
+    runs-on: ubuntu-latest
+
+    env:
+      REDPEN_VERSION: "1.10.1"
+
+    strategy:
+      matrix:
+        dir:
+          - site/ja/guide
+          - site/ja/js
+          - site/ja/lite
+          - site/ja/probability
+          - site/ja/r1
+          - site/ja/tfx
+          - site/ja/tutorials
+          - site/ja/xla
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download RedPen
+        run: |
+          wget https://github.com/redpen-cc/redpen/releases/download/redpen-${REDPEN_VERSION}/redpen-${REDPEN_VERSION}.tar.gz
+          tar xvf redpen-${REDPEN_VERSION}.tar.gz
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.7"
```

Comment:
Make the python version 3.6.

Google is at this version so we can;t test anything beyond 3.6.

Reply:
OK. I fixed it!